content/docs/studio/user-guide/model-registry/add-a-model.md

```diff
@@ -20,39 +20,43 @@ https://www.youtube.com/watch?v=szzv4ZXmYAs
 
    </admon>
 
-3. Enter the path of the model file as follows:
+3. Enter the path to `dvc.yaml` the model will be added to. Adding your model to
+   non-root `dvc.yaml` can be helpful if you develop this ML model in a specific
+   subfolder or if this repo is a
+   [monorepo](/doc/studio/user-guide/projects-and-experiments/configure-a-project#monorepo).
```

Comment:
Is this handled already at the project level? Or are we adding an additional option to pick a subfolder when you add a model?

Reply:
Now in "add a model" user select from views => he's selecting repo + subdir

Reply:
Is there anywhere with context about this decision? Why did we decouple it from the project?